code/modules/jobs/job_types/head_of_personnel.dm

```diff
@@ -13,8 +13,8 @@
 	chat_color = "#7979d3"
 	req_admin_notify = 1
 	minimal_player_age = 10
-	exp_requirements = 360
-	exp_type = EXP_TYPE_CREW
+	exp_requirements = 1200
+	exp_type = EXP_TYPE_SUPPLY
```

Comment:
Why is HOP on supply, I know he leads supply but his job has almost nothing to do with supply

Reply:
Well Service makes even less sense imo. 

Think I'll make require slightly less Command hours than Captain, which would be a more fitting requirement than general Crew hours like espee's suggestion.

Reply:
Why? The HOP is the leader of the service department, he should have normal crew requirements since he assigns jobs, so just playing any job makes the most sense. (Not like pushing crates makes you better at assigning jobs)

Reply:
I'm not a fan since it means someone can do like 20 hours of just Assistant and be qualified. Or another role wholly removed from the station activities like viro.  

On second thought perhaps Service requirement isn't too bad. HOP's never give oversight to Botany making deathnettle, Bartenders serving neurotoxin or the Chef making arnold pizza. Maybe less than 20 hours though.

Reply:
Updated HOP to 10 hours Service